README.md

```diff
@@ -78,6 +75,12 @@ Exposes a multi-tenanted DWN (aka Decentralized Web Node) through a JSON-RPC API
 
 See more in [Storage Options](#storage-options)
 
+## Running online environment
+
+Interested in contributing instantly? You can make your updates directly without cloning in the running CodeSandbox environment.
+
+[![Edit in CodeSandbox](https://assets.codesandbox.io/github/button-edit-lime.svg)](https://codesandbox.io/p/github/TBD54566975/dwn-server/tree/main)
```

Comment:
When I click the link, it takes me to a page that says "Project not found". Maybe there are permissions that need to be updated in Codesandbox?

Reply:
Thanks for the catch. Should be good now. @diehuxx